CMakeLists.txt

```diff
@@ -4,39 +4,46 @@ project(sway C)
 
 set(FALLBACK_CONFIG_DIR "/etc/sway" CACHE PATH
   "Fallback config directory defaults to /etc/sway")
-add_definitions('-DFALLBACK_CONFIG_DIR=\"${FALLBACK_CONFIG_DIR}\"')
+add_definitions(-DFALLBACK_CONFIG_DIR=\"${FALLBACK_CONFIG_DIR}\")
 
-set(CMAKE_C_FLAGS "-g")
+if("${CMAKE_BUILD_TYPE}" STREQUAL "")
+	set(CMAKE_BUILD_TYPE Debug)
+endif()
```

Comment:
I'd like to force debug information so that we get sane backtraces after crashes in the log.